packages/gafl-webapp-service/src/pages/licence-details/licence-for/result-function.js

```diff
@@ -1,6 +1,14 @@
-import { CommonResults } from '../../../constants.js'
+import { CommonResults, Multibuy } from '../../../constants.js'
+import { CheckMultibuyForYou } from '../../../handlers/multibuy-handler.js'
 
 export default async request => {
   const status = await request.cache().helpers.status.getCurrentPermission()
+
+  const isMultibuyForYou = await CheckMultibuyForYou(request)
+
+  if (isMultibuyForYou === true) {
+    return Multibuy.YES
```

Comment:
Perhaps this Multibuy.YES should be MultibuyForYou.YES. To me Multibuy.YES implies that it is multibuy for you or someone else.